src/inc/quic_platform_winuser.h

```diff
@@ -740,8 +740,44 @@ typedef struct CXPLAT_SQE {
 #if DEBUG
     BOOLEAN IsQueued; // Debug flag to catch double queueing.
 #endif
+    HANDLE WcpEvent;      // Manual-reset event for wake packets
```

Comment:
nit: Lets make this name more explicit. `Wcp` is not a well known acronym.
For instance, something along the line of `WaitCompletinoPacketEvent` would be more helpful.